packages/EasyDecision/src/Configurators/AddRestrictedRulesDecisionConfigurator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyDecision\Configurators;
+
+use EonX\EasyDecision\Interfaces\DecisionInterface;
+use EonX\EasyDecision\Interfaces\RestrictedRuleInterface;
+
+final class AddRestrictedRulesDecisionConfigurator extends AbstractConfigurator
```

Comment:
As soon as now we process not only restricted rules in this configurator, we should rename it to something more generic. Maybe `AddRulesDecisionConfigurator` or `AddRulesConfigurator`.